TRLevelReader/Helpers/TR3EntityUtilities.cs

```diff
@@ -359,6 +359,14 @@ public static bool IsAnyPickupType(TR3Entities entity)
                 || IsArtefactPickup(entity);
         }
 
+        public static bool IsVehicleType(TR3Entities entityType)
+        {
+            return entityType == TR3Entities.Quad
+                || entityType == TR3Entities.Kayak
+                || entityType == TR3Entities.UPV
+                || entityType == TR3Entities.MineCart;
```

Comment:
What about the Antarctica boat?

Reply:
Completely forgot about the boat. Now added.